srcpkgs/srb2-data/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'srb2-data'
+pkgname=srb2-data
+version=2.2.6
+revision=1
+archs=noarch
```

Comment:
We no longer have noarch

Could we maybe bundle this as part of the main package? I don't see the need for a separate package.

Reply:
Done

Reply:
Yeah it doesn't even need to be a subpackage, IMO.

Reply:
How do I get a specific distfile to extract to a specific directory?

Reply:
Not really. What you can do is `mv` it manually in `post_extract`.

Reply:
anyway done.